### PR TITLE
Added correct domain for Södertörns Högskola

### PR DESCRIPTION
**Pull request by Serkuz09**
The existing domain contained in the "sh.txt" file leading to Södertörns University College is outdated and leads to nonexistent domain sh.se

### CODE REVIEW

**Comment by philipto** [comment]
@Serkuz09 I am sorry, there must be a misunderstanding. The domain that you call nonexistent exists:
```
$ host sh.se
sh.se has address 185.84.52.190
sh.se mail is handled by 0 sunioffice.mail.protection.outlook.com.
sh.se mail is handled by 10 sh-se.mail.protection.outlook.com.
```

Also, Södertörns Högskola official domain seems not to be suni.se.

Please provide us an independent proof that confirms your claim.

**Comment by Serkuz09** [comment]
Hello and thank you for your reply! While waiting for your reply, I have done some digging as well and there indeed seems to be a misunderstanding. The official domain is still sh.se, which the employees are using. I myself have an email sergej.kuzmin@sh.se on my teachers account. But, on my students account (I am both employed and study at the university), the correct email address is 20seku@suni.se Here are some pictures:
image
image
image
To summarize: the sh.se domain is still active and being used (although not by the students it seems, I should probably check it with the IT department), but the students use the suni.se domain for their emails. This is all very confusing, sorry for any inconvenience! I'll be glad to answer any further questions or to make an inquiry to the IT department. Thanks!

**Comment by philipto** [comment]
@Serkuz09 Thank you for the clarification. Pull request merged.
